rele/subscription.py

```diff
@@ -4,6 +4,10 @@
 from collections.abc import Iterable
 from inspect import getfullargspec, getmodule
 
+from django.conf import settings
+from google.protobuf import duration_pb2
+from google.pubsub_v1.types import RetryPolicy
```

Comment:
Since they have not imported any Google related libraries here I would have probably move the transformation logic into the `client.py` file.